ledgers/ccf/README.md

```diff
@@ -216,13 +216,9 @@ cd ${PDO_SOURCE_ROOT}/build
 make test
 ```
 
-## CCF TP TEE attestation policy
-We briefly describe the attestation verification policy implemented by CCF TP.
-- CCF Governance consortium  registers attestation policy after TP is deployed.
-- Currently, TP supports two policies:
-    - No attestation verification: This policy is used while using PDO enclaves in SGX SIM mode.
-    - Check EPID attestation verification reports generated by IAS: This policy is used while using PDO enclaves in SGX HW mode.
-- One of the two policies must necessarily have been registered by the CCF consortium before any PDO enclave can be registered. There is no default policy.
-CCF TP provides a member-rpc that can be used for registering one of the above two policies. For the second policy above, the consortium
-gets to specify expected MREnclave, basename and IAS public key via the member-rpc.
-- The policy (including expected value of MREnclave) can be changed anytime by the CCF Governance consortium, subject to voting rules of the consortium.
+## CCF TP TEE attestation verification policy
+CCF TP provides two APIs to be used by the CCF Governance consortium to register attestation verification policy that must be satisfied by PDO contract enclaves.
+
+1. The first API `set_attestation_check_flag` is invoked as part of the TP start up scripts to specify whether PDO runs in SGX `HW` mode or SGX `SIM` mode. The flag can be set only once. There is no default value for the flag, and hence must be set explicitly before the TP can accept any `register_encalve` transactions.
+
+2. The second API `set_expected_sgx_measurements` is used whenever the `set_attestation_check_flag` specifies that PDO runs in SGX `HW` mode. In this case, the second API is used to the specify expected `MREnclave` value, and additionally `basename` and the `ias_public_key`. Note that PDO currently supports SGX `HW mode` with EPID attestation. The expected SGX measurements can be updated via the second API, subject to voting rules of the consortium. 
```

Comment:
I think this is mixing API description (which is the objective) with API usage. Separating the two would add clarity.
Namely:
- description: what it does; any default values; "must" or "can" be invoked; how, how many times, and when.
- usage: what scripts use the APIs, and when.

Reply:
agreed. please separate.

and this is not nearly enough detail.

Reply:
The above documentation is supposed to cover "description"  part of what these APIs are. There is no intent to give details about "which scripts use these APIs". AFAIK, we do not have such documentation anywhere inside PDO. Also, the usage of these APIs is not fixed until the upcoming PR on how to use the second API.

Reply:
we really need complete documentation for the pdo tp api. would you please create or find an issue that describes the work to be done and reference it here. after that... i'm fine with this.

Reply:
Thank you. I have updated Issue https://github.com/hyperledger-labs/private-data-objects/issues/264, and added scope for what will we cover to improve documentation (TP's APIs as well as Python scripts used for deployment/management). Will submit a separate PR to address 264.

Reply:
Thank you all the comments, I have updated the README as follows. First provide a description of the attestation verification policy implemented by the TP. Then provide a description of how the 2 APIs are used to set parameters required to implement the described policy. Hope this clarifies.